vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    if MASK_MODE == "semantic":
</pre><p>This is very strange to read. MASK_MODE is set above, and then now it's being compared in a conditional statement. <code>classes</code> is set to <code>None</code>,and then is overwritten if it's <code>semantic</code>.   </p><p> </p><p>I see you're trying to encompass different ways for users to try different combinations, but wouldn't it be easier to show a few examples in action, instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Noted, will make a future change to break this out into multiple cells for clarity.